patches/minecraft/net/minecraft/world/biome/provider/NetherBiomeProvider.java.patch

```diff
@@ -0,0 +1,11 @@
+--- a/net/minecraft/world/biome/provider/NetherBiomeProvider.java
++++ b/net/minecraft/world/biome/provider/NetherBiomeProvider.java
+@@ -77,7 +77,7 @@
+    }
+ 
+    private static NetherBiomeProvider func_235285_d_(long p_235285_0_) {
+-      ImmutableList<Biome> immutablelist = ImmutableList.of(Biomes.field_235254_j_, Biomes.field_235252_ay_, Biomes.field_235253_az_, Biomes.field_235250_aA_, Biomes.field_235251_aB_);
++      ImmutableList<Biome> immutablelist = net.minecraftforge.common.BiomeManager.getNetherBiomes();
```

Comment:
You can simplify this patch and also avoid copying the biome list into the forge class:
`+ immutablelist = net.minecraftforge.common.BiomeManager.getNetherBiomes(immutablelist);`

Reply:
By copying the biome list into the forge class I give modders the ability to remove vanilla biomes if they wish, however.

Reply:
Wouldn't giving modders the ability to remove vanilla biomes bring up the possibility of having mods conflict with each other?

Reply:
That is true - in the case that one modder wishes to expand upon a biome that is then removed or replaced by another. Not sure whether it calls for removing the functionality entirely, however.

The other alternative is that modders who wish to remove or replace a vanilla biome would need to registry replace that biome or the nether dimension.

Reply:
Or use a world type, which is by far a safer route, but that would make incompatibilities with other world type mods.

Reply:
A registry could work, but something like removing integral things from vanilla should probably only be done via ASM so it can't be easily done. In any case, the only mods that should directly control what biomes are to be removed are mods that have their own world gen (i.e. Biomes O' Plenty, just to name one).

Reply:
Should I remove the `removeNetherBiome` method as well, since it will only affect modded biomes and most mods won't need to remove their own biomes or other mod's biomes?

Reply:
> A registry could work, but something like removing integral things from vanilla should probably only be done via ASM so it can't be easily done. 

No.
I agree though, removing things sounds like a bad idea.

Reply:
I personally would remove that method. It's the only criticism I have about this pull request. Other than that, it's great.

Reply:
I've removed that method. Will be gone in my next commit.